monai/transforms/utility/dictionary.py

```diff
@@ -154,6 +155,48 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
         return d
 
 
+class SplitChanneld(MapTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.SplitChannel`.
+    All the input specified by `keys` should be splitted into same count of data.
+
+    """
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        output_postfixes: Sequence[str],
+        channel_dim: Optional[int] = None,
+    ) -> None:
+        """
+        Args:
+            keys: keys of the corresponding items to be transformed.
+                See also: :py:class:`monai.transforms.compose.MapTransform`
+            output_postfixes: the postfixes to construct keys to store split data.
+                for example: if the key of input data is `pred` and split 2 classes, the output
+                data keys will be: pred_(output_postfixes[0]), pred_(output_postfixes[1])
+            channel_dim: which dimension of input image is the channel, default to None
+                to automatically select: if data is numpy array, channel_dim is 0 as
+                `numpy array` is used in the pre transforms, if PyTorch Tensor, channel_dim
+                is 1 as in most of the cases `Tensor` is uses in the post transforms.
+
+        """
+        super().__init__(keys)
+        self.output_postfixes = output_postfixes
```

Comment:
could you add a default `output_postfixes`, to `keys_0`, `keys_1`, ... raise an error if any of the keys already exist  in `data`

Reply:
OK, updated in the latest commit, could you please help review it again?

Thanks.